src/fontra/views/editor/editor.js

```diff
@@ -1574,6 +1583,46 @@ export class EditorController {
     this.sceneController.autoViewBox = false;
   }
 
+  toggleFullscreen() {
+    if (document.fullscreenElement) {
+      if (document.exitFullscreen) {
```

Comment:
Is it possible for `document.fullscreenElement` to exist while `document.exitFullscreen` does not?

Reply:
Right, they go hand in hand, so the inner condition can be removed.